streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/master/graphmanager/GraphManager.java

```diff
@@ -0,0 +1,36 @@
+package org.ray.streaming.runtime.master.graphmanager;
+
+import java.io.Serializable;
+import org.ray.streaming.jobgraph.JobGraph;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+
+/**
+ * Graph manager is one of the important roles of JobMaster. It mainly focuses on graph management.
+ * Such as: 1) build execution graph from job graph
+ *          2) do modifications or operations on graph
+ *          3) query vertex info from graph
+ */
+public interface GraphManager extends Serializable {
```

Comment:
Why `GraphManager` needs to be Serializable

Reply:
I guess this object won't be transferred to other process

Reply:
Fixed.